plugins/poll/config/locales/client.en.yml

```
@@ -35,6 +35,8 @@ en:
           title: "Results will be shown on <strong>vote</strong>."
         closed:
           title: "Results will be shown once <strong>closed</strong>."
+        staff:
+          title: "Results will be shown once <strong>staff</strong>."
```

```suggestion
          title: "Results are only shown to <strong>staff</strong> members."
```

Or something similar?